more_itertools/tests/test_more.py

```diff
@@ -1105,11 +1105,11 @@ def test_basic(self):
         @contextmanager
         def managed():
             before.append(1)
-            yield 1
```

Comment:
I think I caught your meaning and fixed in ce02c10cb8483133513ccea588f2661b7d337d47